src/styles/blocks/header.scss

```diff
@@ -0,0 +1,72 @@
+@use '../utils/mixins' as *;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 50px;
+
+  &__image {
+    height: 40px;
+  }
+
+  a > img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__navigation {
+    display: flex;
+    align-items: center;
+    height: 60px;
+  }
+
+  &__navigation-list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__navigation-item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__navigation-link {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    text-transform: uppercase;
+    text-align: center;
+    display: flex;
+    align-items: center;
+    color: #060b35;
+    height: 60px;
+    text-decoration: none;
+
+    @include transition;
```

Comment:
Ensure that the `transition` mixin is defined in the `mixins` file imported at the top. If it's not defined, this could lead to a compilation error.